cloudify/utils.py

```diff
@@ -753,3 +754,20 @@ def add_plugins_to_uninstall(ctx, plugins_to_uninstall, sequence):
             kwargs={
                 'plugins': plugins_to_uninstall,
                 'delete_managed_plugins': False}))
+
+
+def wait_for(callable_obj,
```

Comment:
why not to put this in the manger only? and maybe this already exists because this is very common

Reply:
I just moved it from a different place and used it. The other functions are in tests and cli, all separated. I think it's out of scope to clean it up and centralize it.